sbatch_scripts/gen_sbatch.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/env python3
+"""
+Generates and submits individual sbatch job for generating simulated data for a given model, and/or for training neural network on simulated data.
+"""
+import argparse
+from argparse import RawDescriptionHelpFormatter
+from pathlib import Path
+import logging
+import subprocess
+import yaml
+import textwrap
+
+# Environment metadata for this script is in the pyproject.toml file
+
+# SBATCH template
+SBATCH_TEMPLATE = """#!/bin/bash
+
+# specifies sbatch resources needed for the job
+
+#SBATCH --mem={mem}
+#SBATCH -J {job_name}
+#SBATCH --time={time}
+#SBATCH --output={output}
+#SBATCH --error={error}
```

Comment:
Let's add an `array` argument to generate several data files here.
```suggestion
#SBATCH --error={error}
#SBATCH --array=1-{array_size}
```
This is called a [job array](https://docs.ccv.brown.edu/oscar/submitting-jobs/array) in slurm lingo.

This will require a few more updates in `create_sbatch_script` and the call to create sbatch_script, and add the arg to the generate arg parser.

Reply:
One needs a range or list of values. If `0` is specified, a single job will run because `0` is a single value. If `array_size` is set to `1`, the directive `--array 1-1` also translates to a single job, but this is more flexible when `array_size>1` jobs are to run.